src/index.html

```diff
@@ -114,6 +114,9 @@ <h2>Calcite Component Examples - WIP !</h2>
         <li>
           <calcite-link href="/demos/calcite-progress.html">calcite-progress</calcite-link>
         </li>
+        <li>
+          <calcite-link href="/demos/calcite-radio.html">calcite-radio</calcite-link>
```

Comment:
Is this needed? `calcite-radio` is a supporting component and wouldn't be used by itself, right?